realsense_camera/src/r200_nodelet.cpp

```diff
@@ -152,6 +153,7 @@ namespace realsense_camera
     {
       rs_set_device_option(rs_device_, RS_OPTION_COLOR_WHITE_BALANCE, config.color_white_balance, 0);
     }
+    rs_apply_depth_control_preset(rs_device_, config.depth_control_preset);
```

Comment:
Since this is R200 specific, could you please have this line under the comment 
// Set R200 specific options